### PR TITLE
Bootstrap React, Material UI and Enzyme for React tests

### PR DESCRIPTION
**Pull request by eko**
Project bootstrap with following technologies:

* **React** 15.4.1 (latest)
* **React-Router** 3.0.0 (latest) in order to brings routing into the React application
* **Webpack** 2.1.0 (latest) in order to compile sources
* **Material UI** 0.16.5 (latest) for the UI
* **Enzyme** 2.7.0 (latest) for testing React components
* **Mocha** 3.2.0 (latest) in order to run tests

I've also added some Github basic templates for issues and pull requests and added `Travis-CI`. @VEBERArnaud can you please enable Travis-CI on the repository with the organization account?

I think we should also implement the `redux` component thereafter to structure application code.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+| Q             | A
+| ------------- | ---
+| Bug fix?      | yes/no
+| New feature?  | yes/no
+| Tests pass?   | yes/no
+| Fixed tickets | #... <!-- #-prefixed issue number(s), if any -->
+| License       | MIT
```
Comment (VEBERArnaud) [inline .github/PULL_REQUEST_TEMPLATE.md]: What do you think about adding a `Breaking change: yes/no` row ?

**Comment by VEBERArnaud** [comment]
Done for Travis-CI > https://travis-ci.org/unikorp/king

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (eko) [inline .github/PULL_REQUEST_TEMPLATE.md]: I thinked about it but I don't see any breaking change that will create issues as we will not expose any public api. Maybe I'm wrong, do you see any breaking change case?

**Comment by eko** [comment]
Thanks dude 🚀

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (jelhaouchi) [inline .github/PULL_REQUEST_TEMPLATE.md]: What about `Doc PR:  yes/no` row ? @eko

Comment (eko) [inline .github/PULL_REQUEST_TEMPLATE.md]: I agree, I will add a documentation section yes, thanks @jelhaouchi

**Pull request merged**